### DISCUSSION TITLE
split menu button - disabled only the primary button

### DISCUSSION THREAD

**Discussion by JohnHavok** [SAP/fundamental-ngx#9384] [Q&A] [answered]
Hello everyone, I find myself in a situation where when using a Split Menu Button, I need to disable the primary button but not the menu associated with it.
The use of a disabled on the split menu button disables all, the button and the menu.
Do you know if it is possible to disable only the primary button ?

**Comment by droshev** [comment, marked as answer]
@JohnHavok this could be a nice improvement. Do you want to open an issue?

**Reply by JohnHavok** [reply]
Sure !
